en/docs/guides/identity-lifecycles/enable-account.md

```diff
@@ -1,71 +1,169 @@
+    !!! info
+        The email template used to send the email notification for
+        account locking is the **AccountLock** template and the template
+        used for account disabling is the **AccountDisable** template. You
+        can edit and customize the email template. For more information on
+        how to do this, see [Customize Automated Emails](../../../guides/tenants/customize-automated-mails)
+.
 
 ---
 
 ## Disable user accounts using SCIM
 
-1. Open the `<IS-HOME>/repository/conf/scim2-schema-extension.config` file and add the following configuration. 
-
-   ```
-   {
-      "attributeURI":"urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:accountDisable",
-      "attributeName":"accountDisable",
-      "dataType":"boolean",
-      "multiValued":"false",
-      "description":"Disable user account.",
-      "required":"false",
-      "caseExact":"false",
-      "mutability":"readwrite",
-      "returned":"default",
-      "uniqueness":"none",
-      "subAttributes":"null",
-      "canonicalValues":[],
-      "referenceTypes":[]
-   }
-   ```
-
-2. Add the `accountDisable` attribute as a sub-attribute of User.
-
-   ```
-   "subAttributes":"verifyEmail askPassword accountDisable employeeNumber costCenter organization division department manager"
-   ```
+1.	Open the `<IS-HOME>/repository/conf/scim2-schema-extension.config` file and add the following configuration. 
+
+	```
+	{
+		"attributeURI":"urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:accountDisable",
+		"attributeName":"accountDisable",
+		"dataType":"boolean",
+		"multiValued":"false",
+		"description":"Disable user account.",
+		"required":"false",
+		"caseExact":"false",
+		"mutability":"readwrite",
+		"returned":"default",
+		"uniqueness":"none",
+		"subAttributes":"null",
+		"canonicalValues":[],
+		"referenceTypes":[]
+	}
+	```
+
+2.	Add the `accountDisable` attribute as a sub-attribute of User.
+
+	```
+	"subAttributes":"verifyEmail askPassword accountDisable employeeNumber costCenter organization division department manager"
+	```
 
 3. Save the file and restart the server. 
 
 Before enabling/disabling users using SCIM, you need to do the following. 
 
-1. [Add claim mapping](insert-link)
+1. [Add claim mapping](../../../guides/dialects/add-claim-mapping/)
 
-2. [Enable account disabling](insert-link)
+2. [Enable account disabling](#disable-an-account)
 
 ### Test it Out 
 
-1. In order to update the status of a user account, we need to obtain the SCIM ID of that particular user. Therefore, we first call the GET users API to get the user details.
+1.	In order to update the status of a user account, we need to obtain the SCIM ID of that particular user. Therefore, we first call the GET users API to get the user details.
 
-   **Request**
+	**Request**
 
-   ``` curl 
-   curl -v -k --user admin:admin 'https://localhost:9443/scim2/Users'
-   ```
-   **Sample**
+	``` curl 
+	curl -v -k --user admin:admin 'https://localhost:9443/scim2/Users'
```

Comment:
the request should have placeholders right? (for the filter, is_host, port and credentials)